draft-tus-resumable-uploads-protocol.md

```diff
@@ -69,9 +72,26 @@ The terms byte sequence, Item, string, sf-binary, sf-boolean, sf-integer, sf-str
 
 The terms client and server are imported from {{HTTP}}.
 
+Upload: A sequence of one or more procedures, uniquely identified by a token chosen by a client.
+
+Procedure: An HTTP message exchange for that can be used for resumable uploads.
+
 # Uploading Overview
 
-The uploading of a file using the Resumable Uploads Protocol consists of multiple procedures:
+The Resumable Uploads Protocol consists of several procedures that rely on HTTP message exchanges. The following procedures are defined:
+
+* Upload Transfer Procedure ({{upload-transfer}})
+* Offset Retrieving Procedure ({{offset-retrieving}})
+* Upload Cancellation Procedure ({{upload-cancellation}})
+
+A single upload is a sequence of one or more procedures. Each upload is uniquely
+identified by a token chosen by a client. The token is carried in the Upload-Token header field; see {{upload-token}}.
+
+The remainder of this section uses an examples of a file upload to illustrate permutations of procedure sequence. Note, however, that HTTP message exchanges use representation data (see {{Section 8.1 of HTTP}}), which means that procedures can apply to many forms of content.
+
+## Example 1: File of known size
+
+In this example, the client attempts to upload an file where the final, experiences an interruption, and then attempts to resume the upload.
```

Comment:
I don't get this sentence.  The final experience is an interruption? Or it finally experiences an interruption?

Reply:
yeah I totally failed here while rewriting the sentence from a version I started with.